clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1893,6 +1899,13 @@ protected void recordProcessedRecordStats(
     }
   }
 
+  @Override
+  protected final void recordAssembledRecordSizeRatio(long size, long currentTimeMs) {
+    final double sizeRatio =
```

Comment:
If the size is unlimited, maybe we shouldn't record `0` all the time as it is a waste of effort.
So just call `recordAssembledRecordSizeRatio` when it is not unlimited.

Reply:
That's a good point. It's pointless to record `0` all the time. I don't think that the value should ever be set to unlimited and the effective default value should be `100MB` though... So there should usually be values for this.

Reply:
Well, for non chunked stores, we don't need to record this metric in theory as it is not our target, right?

Reply:
Yes, this metric should never be recorded for non-chunked stores.